prime-dotnet-webapi/Controllers/AuthorizedUsersController.cs

```diff
@@ -0,0 +1,181 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Auth;
+using Prime.Models;
+using Prime.Models.Api;
+using Prime.Services;
+using Prime.ViewModels;
+using Prime.ViewModels.Parties;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/parties/authorized-users")]
+    [ApiController]
+    [Authorize(Roles = Roles.PrimeEnrollee + "," + Roles.ViewSite)]
+    public class AuthorizedUsersController : ControllerBase
+    {
+        private readonly IAuthorizedUserService _authorizedUserService;
+        private readonly IOrganizationService _organizationService;
+
+        public AuthorizedUsersController(
+            IAuthorizedUserService authorizedUserService,
+            IOrganizationService organizationService)
+        {
+            _authorizedUserService = authorizedUserService;
+            _organizationService = organizationService;
+        }
+
+        // GET: api/parties/authorized-users/5fdd17a6-1797-47a4-97b7-5b27949dd614
+        /// <summary>
+        /// Gets a AuthorizedUser by user ID.
+        /// </summary>
+        /// <param name="userId"></param>
+        [HttpGet("{userId:guid}", Name = nameof(GetAuthorizedUserByUserId))]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(typeof(ApiResultResponse<AuthorizedUserChangeModel>), StatusCodes.Status200OK)]
```

Comment:
I assume this is the wrong ViewModel?

Reply:
Shouldn't this be
```C#
 [ProducesResponseType(typeof(ApiResultResponse<AuthorizedUserViewModel>), StatusCodes.Status200OK)]
 ```

Reply:
Shouldn't this be
```C#
 [ProducesResponseType(typeof(ApiResultResponse<AuthorizedUserViewModel>), StatusCodes.Status200OK)]
 ```

Reply:
@james-hollinger sure is.  Actually all the endpoints except no content have the wrong model.  I'm amazed as a strongly typed language that this generic response doesn't show up as an error, or prevent the build.

Reply:
All this does is generate swagger documentation, there is nothing strongly typed here at all.